tests/performance/tests/rt_exclusive_cpu_pool_scheduling_policy.go

```diff
@@ -0,0 +1,119 @@
+package tests
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	tshelper "github.com/test-network-function/cnfcert-tests-verification/tests/performance/helper"
+	tsparams "github.com/test-network-function/cnfcert-tests-verification/tests/performance/parameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/pod"
+)
+
+var _ = Describe("performance-exclusive-cpu-pool-rt-scheduling-policy",
+	func() {
+
+		BeforeEach(func() {
+			By("Clean namespace before each test")
+			err := namespaces.Clean(tsparams.PerformanceNamespace, globalhelper.APIClient)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		FIt("One pod running in exclusive cpu pool and shared cpu scheduling policy", func() {
+
+			By("Define RT pod")
+			testPod := tshelper.DefineRtPod(tsparams.TestPodName, tsparams.PerformanceNamespace,
+				tsparams.RtImageName, tsparams.TnfTargetPodLabels)
+
+			err := globalhelper.CreateAndWaitUntilPodIsReady(testPod, tsparams.WaitingTime)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Start exclusive-cpu-pool-rt-scheduling-policy test")
+			err = globalhelper.LaunchTests(
+				tsparams.TnfRtExclusiveCPUPoolSchedulingPolicy,
+				globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Verify test case status in Junit and Claim reports")
+			err = globalhelper.ValidateIfReportsAreValid(
+				tsparams.TnfRtExclusiveCPUPoolSchedulingPolicy,
+				globalparameters.TestCasePassed)
+			Expect(err).To(HaveOccurred())
```

Comment:
should be Expect(err).ToNot(HaveOccurred())

Reply:
Done!